subt/docker/run.bash

```diff
@@ -110,6 +110,8 @@ docker run -it \
   -v "/tmp/.X11-unix:/tmp/.X11-unix" \
   -v "/etc/localtime:/etc/localtime:ro" \
   -v "/dev/input:/dev/input" \
+  -v "/home/pavliik/develop/osgar/subt/docker/robotika/ros:/home/developer/subt_solution/src" \
```

Comment:
This mounts a local directory over `src` directory in the catkin_workspace. The original directory contained `subt` and `subt_seed` directories. I don't see how this works. We still need ros_proxy_node.cc that should be copied over `subt_solution/subt_seed/subt_seed_node.cc`. I don't follow.